Dockerfile

```
@@ -6,8 +6,8 @@ FROM rocker/r-ver:4.4.0
 LABEL org.opencontainers.image.authors="CCDL ccdl@alexslemonade.org"
 LABEL org.opencontainers.image.source="https://github.com/AlexsLemonade/spellcheck/tree/main"
 
-# install the spelling and tidyr packages from CRAN
-RUN Rscript -e "install.packages(c('readr', 'spelling', 'tidyr'))"
+# install R package dependencies from CRAN
+RUN Rscript -e "install.packages(c('spelling', 'readr', 'purrr', 'dplyr', 'tidyr'))"
```

alphabetical imports
```suggestion
RUN Rscript -e "install.packages(c('dplyr', 'purrr', 'readr', 'spelling', 'tidyr'))"
```